.config/ags/variables.js

```diff
@@ -14,7 +16,7 @@ export const showColorScheme = Variable(false, {})
 globalThis['openMusicControls'] = showMusicControls;
 globalThis['openColorScheme'] = showColorScheme;
 globalThis['mpris'] = Mpris;
-
+globalThis['getString'] = getString
```

Comment:
Since getString is a global variable, maybe remove import statements for it from the other files? I've seen one or two redundant ones

Reply:
Modifications have been made